### DISCUSSION TITLE
Make "gap" automatically use "grid-gap" on Safari

### DISCUSSION THREAD

**Discussion by w3bdesign** [tailwindlabs/tailwindcss#3725] [Ideas]
There is an issue where `gap-` is not supported on older Safari devices. 

As a result, there is a need to use `grid-gap: x px` with CSS in order to get it to work. 

I suggest that this gets fixed in one way or another so that there is no need for a manual CSS fix.

**Comment by adamwathan** [comment]
Hey! We used to include `grid-gap` in Tailwind v1.x but we don't support old Safari anymore as of Tailwind v2.0. The regular `gap` property works as of Safari 12 on both macOS and iOS.
